src/styles/untils/_variables.scss

```
@@ -0,0 +1,21 @@
+$watchSize: 400; /*px*/
+$TimeSpeed: 20; /* 1 - default*/
+
+$wColor-First: #e15443;
+$wColor-Second: #485268;
+
+$BGC-Upper: #fbefee;
+$BGC-Lower: $wColor-First;
+
+$BGC-Power-Lower: -50;
+$BGC-Power-Upper: 20;
+$BGC-Deg: -30;
+
+/* #region Default Values */
+
+$fontwatchSize: $watchSize * 0.06; /*dont change*/
+$shadowSize: $watchSize * 0.03; /*dont change*/
+$second: 60 / $TimeSpeed; /*dont change*/
+$minute: $second * 60; /*dont change*/
+$hour: $minute * 12; /*dont change*/
+/* #endregion */
```

```suggestion
$watchSize: 400; /*px*/
$TimeSpeed: 20; /* 1 - default*/

$wColor-First: #e15443;
$wColor-Second: #485268;

$BGC-Upper: #fbefee;
$BGC-Lower: $wColor-First;

$BGC-Power-Lower: -50;
$BGC-Power-Upper: 20;
$BGC-Deg: -30;

/* #region Default Values */

$fontwatchSize: $watchSize * 0.06; 
$shadowSize: $watchSize * 0.03; 
$second: 60 / $TimeSpeed; 
$minute: $second * 60; 
$hour: $minute * 12; 
```
you can remove these comments